src/printSuccess.go

```diff
@@ -3,7 +3,7 @@ package main
 import "fmt"
 
 var serviceCmds = map[string]string{
-	"typescript": "npm run services",
+	"typescript": "npm run dev",
```

Comment:
for consistency in case we wanna have a dropdown of examples, we probably need to make sure all examples have either npm run dev or npm run services